ajax/libs/leaflet-groupedlayercontrol/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "leaflet-groupedlayercontrol",
+  "version": "0.6.0",
+  "description": "A Leaflet layer control with support for grouping overlays together",
+  "filename": "leaflet.groupedlayercontrol.min.js",
+  "repository": {
```

Comment:
please change repository url to `https://github.com/ismyrnow/leaflet-groupedlayercontrol`